Kernel/Arch/aarch64/Processor.h

```diff
@@ -260,6 +260,8 @@ class Processor {
     FlatPtr init_context(Thread& thread, bool leave_crit);
     static ErrorOr<Vector<FlatPtr, 32>> capture_stack_trace(Thread& thread, size_t max_frames = 0);
 
+    static StringView platform_string();
```

Comment:
This change to add platform_string() to aarch64/Processor.h seems independent and should be its own commit